App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
+import Login from './Login';
+
+const App = () => (
+  <Router>
```

Comment:
Consider adding a Switch component from 'react-router-dom' to wrap your Route and Redirect components. This will ensure that only the first matching route is rendered, which can improve performance and prevent potential rendering issues.